packages/babel-plugin-polyfill-es-shims/src/index.ts

```diff
@@ -30,10 +29,15 @@ export default defineProvider<{||}>(function ({
   });
 
   function createDescIterator(
-    cb: (Descriptor, Utils, Object) => void,
-    instance?: (MetaDescriptor, *, Utils, Object) => void,
+    cb: (desc: Descriptor, utils: Utils, path: any) => void,
+    instance?: (
+      meta: MetaDescriptor,
+      resolved: any,
+      utils: Utils,
+      path: any,
```

Comment:
Per [prev discussion](https://github.com/babel/babel-polyfills/pull/121#discussion_r837907615), I left this as `any` for now as changing it to just `NodePath` (with no generic or further specification) adds ~7 type errors